### PR TITLE
chore(docker): docker-compose mongo service local dev

### PR DESCRIPTION
**Pull request by gordonpn**
* add example of making an api call to the server from the client
  * Note: don't use the fetch api but something like [react-query](https://github.com/tannerlinsley/react-query) in the future
* add example of init script to seed mongodb with data
* enable cors on server
* add mongo-express to visually browse the database from a browser

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-k0uuva Dockerfiles and docker-compose](https://app.clickup.com/t/k0uuva)

File: client/src/components/Hero.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
-);
+export const Hero = ({ title }) => {
+  const [subtitle, setSubtitle] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const baseUrl = process.env.NEXT_PUBLIC_API_URL;
+      fetch(baseUrl).then((data) => {
+        setSubtitle(`Request to backend :9090 status: ${data.status}`);
+        console.log(data);
```
Comment (nasjuice) [inline client/src/components/Hero.js:12]: side note: shouldn't we avoid console.log and use a front-end logger

Comment (gordonpn) [reply client/src/components/Hero.js:12]: You're right, let me remove that

File: Makefile [hunk 2]
```diff
@@ -11,7 +11,7 @@ help: ## Show this help
 	@egrep -h '\s##\s' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[33m%-20s\033[0m %s\n", $$1, $$2}'
 
 run: ## Start up both client and server containers
-	docker-compose up --build
+	docker-compose up --build --force-recreate
```
Comment (nasjuice) [inline Makefile:14]: why do we need this?

Comment (gordonpn) [reply Makefile:14]: Docker would keep reusing cached layers of package.json even though I had just installed packages, it got a bit annoying

**Review by nasjuice — DISMISSED** [review]

**Review by Mackbellemore — APPROVED** [review]
:waffle:

File: docker-compose.yml [hunk 3]
```diff
@@ -13,6 +13,8 @@ services:
       - ./client:/app
       - /app/.next
```
Comment (Mackbellemore) [inline docker-compose.yml:14]: before you merge is this needed?

**Review by Mackbellemore — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (Mackbellemore) [reply docker-compose.yml:14]: can you try without it

Comment (Mackbellemore) [reply docker-compose.yml:14]: ah wtv fock it

**Review by Mackbellemore — APPROVED** [review]

**Review by Mackbellemore — APPROVED** [review]

**Review by nasjuice — APPROVED** [review]

**Pull request merged**